build_scripts/build_usd.py

```diff
@@ -2367,6 +2380,8 @@ def __init__(self, args):
         self.buildOIIO = ((args.build_oiio or (self.buildUsdImaging
                                                and self.buildTests))
                           and not embedded)
+        if MacOS():
+            self.buildImageIO = args.build_imageio and not self.buildOIIO
```

Comment:
Can we change this so that ImageIO and OpenImageIO don't turn each other off?  i.e. get rid of the "and not self.buildOIIO" here...

Reply:
Sure, would you like me to document somewhere that they should ideally not be used together? Or just let Hio pick one itself at runtime?

Reply:
I ended up going with your suggestion below to make ImageIO take precedence.